src/ia/scss/_vars.scss

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
@jdorweiler I was thinking since other stuff might go in here it's good to separate them by a comment. But I guess the only ones here are colors at the moment.